tests/plugin.spec.ts

```diff
@@ -292,6 +295,61 @@ describe('plugin', () => {
     expect(result.data?.getNode).toEqual({ __typename: 'AddsNode', id: 'AddsNode:abc', name: 'test' })
   })
 
+  it('has an onFieldDefinition / onArgDefinition / onInputFieldDefinition option, which receives the field metadata, and can modify the field', async () => {
+    //
+    const schema = makeSchema({
+      outputs: false,
+      features: {
+        abstractTypeStrategies: {
+          __typename: true,
+        },
+      },
+      types: [
+        queryField('ok', {
+          type: 'Boolean',
+          // @ts-ignore
+          list: true,
+          args: {
+            // @ts-ignore
+            filter: booleanArg({ list: true }),
+            input: inputObjectType({
+              name: 'SomeType',
+              definition(t) {
+                // @ts-ignore
+                t.boolean('inputField', { list: true })
+              },
+            }),
+          },
+          resolve: () => [true],
+        }),
+      ],
+      plugins: [
+        plugin({
+          name: 'Node',
+          onFieldDefinition(field, config) {
```

Comment:
Could it make sense to call this `onOutputFieldDefinition`?

Reply:
Good call. Done.